extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -251,6 +252,11 @@ export class AmpA4A extends AMP.BaseElement {
     /** @private {?ArrayBuffer} */
     this.creativeBody_ = null;
 
+    /** @private {string} The random subdomain to load SafeFrame from */
+    this.safeFrameSubdomain_ = Services.cryptoFor(
```

Comment:
Looks like the polyfill is only loaded if a method inside of the crypto lib calls `loadpolyfill_()`. Instead of doing that, `getSecureRandomString` falls back to Math.random if `this.win_.crypto` isn't available.

Reply:
done